h2o-algos/build.gradle

```diff
@@ -12,9 +12,29 @@ dependencies {
 
   // Jama dependencies
   compile "gov.nist.math:jama:1.0.3"
+
   // netlib-java / MTJ dependencies
-  compile 'com.github.fommil.netlib:all:1.1.2'
-  compile('com.googlecode.matrix-toolkits-java:mtj:1.0.4') { exclude group: 'com.github.fommil.netlib' }
+
+  // Manually define all dependencies in com.github.fommil.netlib:all:1.1.2 to make Databricks maven resolver happy
+  compile('com.github.fommil.netlib:core:1.1.2')
```

Comment:
DBC is not happy with `all` artifact. We just manually specify everything what is inside `all`. Manual, but working solution.